daemon/volumes.go

```diff
@@ -250,6 +250,9 @@ func (daemon *Daemon) verifyOldVolumesInfo(container *Container) error {
 	}
 
 	for destination, hostPath := range vols.Volumes {
+		if container.MountPoints == nil {
```

Comment:
@duglin  It's more reasonable to add into `addLocalMountPoint` to catch all possible calls to it for future use.  And just for in case, add this into `addMountPointWithVolume()` is also reasonable. I'll update